docs/install/collecting-metrics/collector.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+  namespace: 
```

Comment:
`metrics`?

Reply:
Yes, fixed. (I was trying to copy things from one machine to another to test the hand-rolled setup extracted from the operator configuration I'm using on an existing cluster.)